source/Playnite/Controllers/GenericGameController.cs

```diff
@@ -20,26 +21,43 @@ public class GenericGameController : BaseGameController
         protected Stopwatch stopWatch;
         protected ProcessMonitor procMon;
         private GameDatabase database;
+        private static ILogger logger = LogManager.GetLogger();
 
-        public GenericGameController(GameDatabase db, Game game) : base (game)
+        public GenericGameController(GameDatabase db, Game game) : base(game)
         {
             database = db;
         }
 
         public override void Play()
-        {            
+        {
             if (Game.PlayAction == null)
             {
                 throw new Exception("Cannot start game without play action.");
             }
 
-            var playAction = Game.PlayAction.ExpandVariables(Game);
+            var gameClone = Game.GetClone();
+            var playAction = gameClone.PlayAction;
+            var emulators = database.Emulators.ToList();

```

Comment:
Why are you creating copy list of emulators? I don't see it use anywhere, but in one place, and I don't see a reason for this. Please correct me if I'm wrong.

Reply:
I did not add ToList() call. It was already there. I guess because the GameActionActivator.GetGameActionEmulatorConfig(playAction, emulators) method needs an List<Emulator> instance as second parameter

Reply:
Hmm, ok then.